.github/workflows/publish-cloud.yml

```diff
@@ -63,24 +65,27 @@ jobs:
         with:
           MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
           MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
-          COMMIT_MESSAGE: "Bump ${{ inputs.PACKAGE_VERSION }}"
+          COMMIT_MESSAGE: "Bump ${{ env.PACKAGE_VERSION || inputs.version }}"
```

Comment:
The `env.PACKAGE_VERSION` should be calculated on `inputs.version` (if needed)  and be consistent over the whole workflow